### PR TITLE
document channel concept

### PR DESCRIPTION
**Pull request by swwilson-bsi**

### CODE REVIEW

File: core/imodel-bridge/src/IModelBridge.ts [hunk 1]
```diff
@@ -28,34 +28,35 @@ export abstract class IModelBridge {
   }
 
   /** This is only called the first time this source data is synchronized.  Allows the bridge to perform any steps after the Job Subject has been created.  It
-   * must call synchronizer.recordDocument on the source data.
+   * must call synchronizer.recordDocument on the source data. Called in the [Repository channel]($docs/learning/backend/channels).
```
Comment (pmconne) [inline core/imodel-bridge/src/IModelBridge.ts:31]: @swwilson-bsi Are these links intended to point to Channel.md directly, or does something map `channels` to that file?

Comment (swwilson-bsi) [reply core/imodel-bridge/src/IModelBridge.ts:31]: It is supposed to link to channels.md.

Comment (pmconne) [reply core/imodel-bridge/src/IModelBridge.ts:31]: Do you know if doc links are case-sensitive ("Channel" vs "channel")? @williamkbentley?

Comment (williamkbentley) [reply core/imodel-bridge/src/IModelBridge.ts]: The $docs link syntax is case insensitive.

Comment (swwilson-bsi) [reply core/imodel-bridge/src/IModelBridge.ts]: Fixed.

**Review by pmconne — APPROVED** [review]

**Review by scsewall — COMMENTED** [review]
We will eventually want the notion of sub-channels, so not sure about the "do not nest" comment.

**Comment by swwilson-bsi** [comment]
> We will eventually want the notion of sub-channels, so not sure about the "do not nest" comment.

We could relax that restriction in the future. The current implementation of getChannelOfElement is optimized based on the assumption of no nesting. If we add nesting, then we will need a way to look up channel membership without too much cost.

**Review by c-macdonald — APPROVED** [review]

**Review by abeesh — APPROVED** [review]

**Pull request merged**